scripts/exgdas_global_marine_analysis_vrfy.py

```diff
@@ -41,6 +41,61 @@
 HOMEgfs = os.getenv('HOMEgfs')
 
 
+#######################################
+# recentering error
+#######################################
+string = 'recentering_error'
```

Comment:
"string" is a little generic

Reply:
removed string, no clue what the other stuff means @AndrewEichmann-NOAA